www/js/constants.js

```diff
@@ -0,0 +1,8 @@
+angular.module('bookingz.constants', [])
+  .constant('APP_VERSION', '0.0.5')
+  .constant('API_URL_STAGING', 'https://cal-bookingz.herokuapp.com')
+  .constant('API_URL', 'https://komboxrum.herokuapp.com')
+  .constant('API_URL_DEVELOP', 'http://localhost:3000')
+  .constant('SHOW_RESOURCE_NAME', 'false')
+  .constant('SHOW_DATE', 'false')
+  .constant('APP_VERSION', '0.1.3-alpha');
```

Comment:
@tochman `APP_VERSION` is defined twice with 2 different values.. Which one are we using?